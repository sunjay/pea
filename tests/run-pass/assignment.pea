fn foo() {
    println!(b"foo");
}

fn bar() {
    println!(b"bar");
}

fn main() {
    let a = 3;
    println!(a);
    a = a + 4;
    println!(a);

    let b = foo;
    b();
    b = bar;
    b();
}
