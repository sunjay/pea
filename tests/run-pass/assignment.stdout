3
7
foo
bar
